validator/src/main.rs

```diff
@@ -114,6 +114,7 @@ const DEFAULT_MIN_SNAPSHOT_DOWNLOAD_SPEED: u64 = 10485760;
 // The maximum times of snapshot download abort and retry
 const MAX_SNAPSHOT_DOWNLOAD_ABORT: u32 = 5;
 const MILLIS_PER_SECOND: u64 = 1000;
+const DEFAULT_PREALLOCATED_BUNDLE_COST: u64 = 1000000;
```

Comment:
arbs are ~150k compute units, so i think we should set default higher to like 2M or 3M

Reply:
make sense